partcad/src/partcad/ai_feature_file.py

```
@@ -0,0 +1,92 @@
+        content_inserts = []
+
+        def include_content(match):
+            location = match.group(2)
+            if (location.startswith('"') and location.endswith('"')) or (
+                location.startswith("'") and location.endswith("'")
+            ):
+                location = location[1:-1]
+            try:
+                content_inserts.append(AiContentFile(location))
+                return "CONTENT_INSERTED_HERE"
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)